cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsDetailsActions/PipelineDetailsDetailsActions.scss

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+.pipeline-details-details-actions {
+  .pipeline-details-container {
+    .pipeline-details-btn {
+      .icon-info-circle {
+        color: initial;
```

Comment:
nit: You might want to straight away use the right color here.

`initial` can be avoided since,
- is specific for browser (like a hard reset from its property definition table)
- IE doesn't support initial. Not that we care about it but if we know for sure if this is going to be a problem might as well avoid it.